libkbfs/kbfs_ops_concur_test.go

```diff
@@ -690,7 +690,7 @@ func TestKBFSOpsConcurBlockSyncTruncate(t *testing.T) {
 
 	wg.Wait()
 
-	// Do this in the main goroutine since t isn't goroutine safe,
+	// Do this in the main goroutine since it isn't goroutine safe,
```

Comment:
If you feel like it, may as well apply all the suggestions below to this test too.